packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts

```diff
@@ -14,17 +15,29 @@ const {
 } = swapEgressScheduledMock;
 
 jest.mock('axios', () => ({
-  post: jest.fn(() =>
-    Promise.resolve({ data: environment({ egressFee: '0x777777' }) }),
-  ),
+  post: jest.fn((url, data) => {
+    if (data.method === 'cf_environment') {
+      return Promise.resolve({ data: environment({ egressFee: '0x777777' }) });
+    }
+
+    if (data.method === 'cf_swap_rate') {
+      return Promise.resolve({
+        data: swapRate({
+          output: `0x${(BigInt(data.params[2]) * 3n).toString(16)}`,
```

Comment:
why `* 3n`?

Reply:
just a random rate that could be the rate of the pool